scripts/add-fields-for-users.sql

```diff
@@ -0,0 +1,12 @@
+use mydb;
+
+CREATE TABLE users (
+    id char(36) PRIMARY KEY,
+    user_id VARCHAR(255) NOT NULL,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    sex VARCHAR(50),
+    email VARCHAR(255) NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
```

Comment:
I do not want to edit the existing SQL file.
The purpose is to create a new file and modify table to add new columns using `ALTER TABLE`.
Example: https://www.w3schools.com/sql/sql_alter.asp